pkg/freeze/crio/pause_test.go

```diff
@@ -9,14 +9,17 @@ import (
 	"knative.dev/container-freezer/pkg/freeze/test"
 )
 
-func runServerAndCreateProvider(ctx context.Context) (*CrioCRI, *test.CRIServer, *test.CrioServer, error) {
+func runServerAndCreateProvider(ctx context.Context, t *testing.T) (*CrioCRI, *test.CRIServer, *test.CrioServer, error) {
 	criSocketPath := test.GetRandomSocketPath()
 	criServer := test.NewCriRuntimeServer()
 	go test.RunCriServer(criServer, criSocketPath)
+	t.Logf("Create cri sock path:%s", criSocketPath)
```

Comment:
not sure we need to log this or the crio one for a test unless the test fails, so maybe it would be better to add this to the error (if it's necessary)?

Reply:
The ut will fail sometimes, add log for better find it out the reason, actually not need it, I delete it.